### DISCUSSION TITLE
GitHub Shop

### DISCUSSION THREAD

**Discussion by medk0v** [community/community#44206] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Is it working?
Total ignore of my emails sent to shop@github.com
Made two orders and got a refund for first(dont know why, products are still available) and no tracking number for second.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! 

I've flagged this to our team - would you mind reaching out to [support@imprintengine.com](mailto:support@imprintengine.com) please? They'll help get this addressed. 

Cheers!
